packages/pv-create-component/lib/templates/jsTemplate.js

```diff
@@ -0,0 +1,63 @@
+module.exports = function({constructorName, componentName}) {
+
+  return (
+`import { Component } from "Core/Component";
+import { ACTIVE } from "Helper/cssClasses";
+import { addClass, removeClass} from "Helper/domHelper";
+
+/**
+ * Custom Element to .. @TODO
+ */
+class ${constructorName} extends Component {
+
+  constructor() {
+    super({
+
+      ui: {
+        input: ".${componentName}__input :-one",
+        buttons: ".${componentName}__label",
+      },
+
+      events: [
+        {
+          event: "keyup",
+          target: "$input",
+          handler: "handleInputChange",
+        },
+      ],
+
+      props: {
+        score: {
+          type: "number",
+          required: false,
+          default: 1,
+        }
+      }
+    });
+  }
+
+  /**
+   * @override
+   */
+  connectedCallback() {
```

Comment:
maybe afterComponentRender()-Hook would be a good example

Reply:
updated